src/main/java/juja/microservices/gamification/slackbot/dao/RestUserRepository.java

```diff
@@ -2,24 +2,28 @@
 
 import juja.microservices.gamification.slackbot.exceptions.GamificationExchangeException;
 import juja.microservices.gamification.slackbot.model.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.http.*;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
-import org.springframework.web.util.UriComponentsBuilder;
 
 import javax.inject.Inject;
 import java.util.HashMap;
-import java.util.Map;
 
 /**
  * Created by Artem
  */
 
-
+@PropertySource("application.properties")
 public class RestUserRepository implements UserRepository {
 
     private final RestTemplate restTemplate;
-    private final String URL_GET_USER = "/users/search";
+
+    @Value("${user.baseURL}")
+    private String BASE_URL;

```

Comment:
according to code convention BASE_URL means constant. if you want use BASE_URL you have to use "private final String"